preview_generator/preview/builder/image__pillow.py

```diff
@@ -2,13 +2,21 @@
 
 from io import BytesIO
 import logging
+
+import PIL
 from PIL import Image
 import typing
 
 from preview_generator.preview.generic_preview import ImagePreviewBuilder
 from preview_generator.utils import compute_resize_dims
 from preview_generator.utils import ImgDims
 
+JPEG_OPTIMIZE = True
+JPEG_QUALITY = 95
+JPEG_PROGRESSIVE = True
+JPEG_RESAMPLE = PIL.Image.BILINEAR
+TRANSPARENCY_MODES = ['RGBA', 'LA']
```

Comment:
where are these string values from? Please add a link / a reference explaining what this transparency modes are

Reply:
done.

Reply:
after checking : https://pillow.readthedocs.io/en/latest/handbook/concepts.html#modes 
It seems to have also `RGBa` as transparency mode, we can support it but i do not know yet how to test if this really work.

Reply:
added support for RGBa using convertion. Don't know if in normal usage Pillow can convert to RGBa by default but this case is handled.